airbyte-cdk/python/airbyte_cdk/utils/__init__.py

```diff
@@ -5,5 +5,7 @@
 from .is_cloud_environment import is_cloud_environment
 from .schema_inferrer import SchemaInferrer
 from .traced_exception import AirbyteTracedException
+from .stream_status_utils import as_airbyte_message
+from .spec_schema_transformations import resolve_refs
 
-__all__ = ["AirbyteTracedException", "SchemaInferrer", "is_cloud_environment"]
+__all__ = ["AirbyteTracedException", "SchemaInferrer", "is_cloud_environment", "as_airbyte_message", "resolve_refs"]
```

Comment:
`resolved_refs` is used by a few destinations so it should be part of the public interface. We can revisit separately if we don't want it to be